content/ja/docs/topics/advanced.md

```diff
@@ -50,23 +50,23 @@ bash では、これは `renderer1 | renderer2 | renderer3` のように単純
 `kustomize` をポストレンダラーとして使用する例は
 [here](https://github.com/thomastaylor312/advanced-helm-demos/tree/master/post-render)で見ることができます。
 
-### Caveats
-When using post renderers, there are several important things to keep in mind.
-The most important of these is that when using a post-renderer, all people
-modifying that release **MUST** use the same renderer in order to have
-repeatable builds. This feature is purposefully built to allow any user to
-switch out which renderer they are using or to stop using a renderer, but this
-should be done deliberately to avoid accidental modification or data loss.
-
-One other important note is around security. If you are using a post-renderer,
-you should ensure it is coming from a reliable source (as is the case for any
-other arbitrary executable). Using non-trusted or non-verified renderers is NOT
-recommended as they have full access to rendered templates, which often contain
-secret data.
```

Comment:
> as they have full access to rendered templates, which often contain secret data.

This is the reason to use trusted renderers. Can you confirm that this reason is included in your translation? Thanks!